ui/src/app/base/sagas/websockets.js

```diff
@@ -257,12 +257,24 @@ export function* handleMessage(socketChannel, socketClient) {
           type: `${action.type}_ERROR`,
           error,
         });
+        // Temporarily send both formats of action names.
+        yield put({
+          meta: { item },
+          type: `${action.type}Error`,
```

Comment:
The format of the actions needs to match the key of the name used to create the reducer/action.